tests/Unit/CategoriesTest.php

```diff
@@ -7,41 +7,136 @@
 class CategoriesTest extends TestDataSetup
 {
     /**
-     * A basic test example.
-     *
-     * @return void
+     * Total Test Cases: 10
      */
+
+    /**
+     * Positive Test Cases: 6
+     */
+
+    public function test_index_returns_expected_structure()
+    {
+        $this->get('/api/categories')
+                ->assertStatus(200)
+                ->assertJsonStructure([ 
+                    'length',
+                    'data' => [
+                        '*' => [
+                            'id',
+                            'parent_id',
+                            'name',
+                            'description',
+                            'access_level',
+                            'created_at',
+                            'updated_at'
+                        ]
+                    ]
+                ]);
+    }
+
+    public function test_index_returns_expected_length()
+    {
+        $content = $this->get('/api/categories')->decodeResponseJson();
+        $this->assertEquals($content['length'], count(\App\Category::all()));
+    }
+
     public function test_show_returns_expected_structure()
     {
         $this->get('/api/categories/' . $this->category1->id)
+                ->assertStatus(200)
                 ->assertJsonStructure([
-                        'id',
-                        'parent_id',
-                        'name',
-                        'description',
-                        'access_level',
-                        'created_at',
-                        'updated_at'
+                    'id',
+                    'parent_id',
+                    'name',
+                    'description',
+                    'access_level',
+                    'created_at',
+                    'updated_at'
                 ]);
     }
 
-    public function test_index_returns_all()
+    public function test_store_can_persist_data()
     {
-        $content = $this->get('/api/categories')->decodeResponseJson();
+        $category = [
+            'name' => 'test category',
+            'description' => 'test category desc',
+            'access_level' => 'F'
+        ];
 
-        $this->assertEquals($content['length'], count(\App\Category::all()));
+        $this->post('/api/categories', $category)
+                ->assertStatus(201)
+                ->assertJsonFragment($category);
     }
 
-    public function test_that_store_method_can_persist_data()
+    public function test_update_can_persist_data()
     {
         $category = [
-            'name' => 'test category1',
-            'description' => 'desc',
+            'name' => 'test category',
+            'description' => 'test category desc',
+            'access_level' => 'F'
+        ];
+        $response = $this->post('/api/categories', $category)->decodeResponseJson();
```

Comment:
Here you are creating a category first for the purpose of updating later. Problem is - If there is any issue in `create` method but no issue in `update` method, then also update test case will fail. Since these are unit tests, you can not put two different functionalities together.